specs/protocol/configurability.md

```diff
@@ -283,6 +286,21 @@ contracts deployed on layer 1.<br/>
 **Requirement:** Disabled<br/>
 **Notes:** Simple clear restriction.<br/>
 
+### [Operator Fee Scalar](exec-engine.md#operator-fees)
+
+**Description:** Operator fee scalar -- used to calculate the operator fee<br/>
+**Administrator:** [Operator Fee Manager](#operator-fee-manager)<br/>
```

Comment:
See comment [below](https://github.com/ethereum-optimism/specs/pull/382/files#r1938081685), this should be System Config Owner

Reply:
Done